@kiva/kv-shop/src/basketItems.ts

```diff
@@ -1,16 +1,19 @@
-import { gql } from '@apollo/client/core';
+import { gql, ApolloClient } from '@apollo/client/core';
 import numeral from 'numeral';
-import { getBasketID } from './basket';
+import { GraphQLErrors } from '@apollo/client/errors';
+import { getBasketID, hasBasketExpired, handleInvalidBasketForDonation } from './basket';
 import { parseShopError } from './shopError';
 
 export interface SetTipDonationOptions {
 	amount: string | number,
-	apollo: any,
+	apollo: ApolloClient<any>,
```

Comment:
Actually, this could be `ApolloClient<NormalizedCacheObject>` to get around the linting error.